src/extras/core/Path.js

```diff
@@ -168,7 +168,13 @@ THREE.Path.prototype.ellipse = function ( aX, aY, xRadius, yRadius,
 THREE.Path.prototype.absellipse = function ( aX, aY, xRadius, yRadius,
 									  aStartAngle, aEndAngle, aClockwise, aRotation ) {
 
-	var args = Array.prototype.slice.call( arguments );
+	var args = [
+		aX, aY,
+		xRadius, yRadius,
+		aStartAngle, aEndAngle,
+		aClockwise,
+		aRotation || 0 // aRotation is optional.
+	];
```

Comment:
@WestLangley I'm not sure I'm still allowed to comment on this... I think it would be better to get rid of `args` and replace
```javascript
this.actions.push( { action: THREE.PathActions.ELLIPSE, args: args } );
```
with something like this if possible
```javascript
this.actions.push( { action: THREE.PathActions.ELLIPSE, curve: curve} );
```